zScanner/ViewModels/Documents/DocumentViewModel.swift

```diff
@@ -120,6 +126,17 @@ class DocumentViewModel {
             .disposed(by: disposeBag)
     }
     
+    private let realm = try! RealmDatabase()
+    
+    func saveTimeOfSeding() {
+        let documentModel = database.loadObject(DocumentDatabaseModel.self, withId: self.document.id)
+        guard let document = documentModel else { return }
+        try! realm.write {
+            document.sent = Date()
+         }
+        database.saveObject(document)
+    }
```

Comment:
To save the hassle with storing extra `DocumentDatabaseModel `, you can save the success time to the `UploadStatus`, there is already code for storing it (line 92) , so you just need to add one more parameter to the `DocumentUploadStatusDatabaseModel()`